### PR TITLE
Allow to save screenshots in folders

### PR DESCRIPTION
**Pull request by sebastienrouif**
Adds the ability to add a path to every screenshot test,
I updated the sample project to show an example

### CODE REVIEW

File: sample/app/src/androidTest/screenshots/basicActivityView.png
Comment (sebastienrouif) [inline sample/app/src/androidTest/screenshots/basicActivityView.png:1]: I used a pixel5 emulator but the screen is bigger

Comment (sebastienrouif) [reply sample/app/src/androidTest/screenshots/basicActivityView.png:1]: @rharter should I update anything to get this moving ?

**Comment by rharter** [comment]
Sorry, I'm travelling to speak at KotlinConf this week so AFK for a bit.
I'll take a closer look as soon as I can.

File: sample/app/src/androidTest/screenshots/basicActivityView.png
Comment (sebastienrouif) [reply sample/app/src/androidTest/screenshots/basicActivityView.png:1]: @rharter I'll most likely need to take screenshots again as I don't think I had the right config

**Pull request merged**